File: sdkwork-ai-sdk-csharp/Models/RerankResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class RerankResponse
    {
        public string? Id { get; set; }
        public string? Object { get; set; }
        public string? Model { get; set; }
        public List<Result>? Results { get; set; }
        public Usage? Usage { get; set; }
    }
}

File: sdkwork-ai-sdk-typescript/src/types/rerank-response.ts
import type { Result } from './result';
import type { Usage } from './usage';

/** 重排响应 */
export interface RerankResponse {
  /** 请求ID */
  id?: string;
  /** 对象类型 */
  object?: string;
  /** 使用的模型 */
  model?: string;
  /** 重排结果列表 */
  results?: Result[];
  /** token用量 */
  usage?: Usage;
}
